app07_youtube_app/lib/models/video.dart

```diff
@@ -0,0 +1,25 @@
+class Video {
+  String id;
+  String title;
+  String description;
+  String imageUrl;
+  String channel;
+
+  Video({
+    required this.id,
+    required this.title,
+    required this.description,
+    required this.imageUrl,
+    required this.channel,
+  });
```

Comment:
Data classes must be immutable so you can guarantee that the value hasn't changed down the road. 
To do that, use the keyword `final` before every class member.

```suggestion
  String id;
  final String title;
  final String description;
  final String imageUrl;
  final String channel;

    const  Video({
    required this.id,
    required this.title,
    required this.description,
    required this.imageUrl,
    required this.channel,
  });
```